scale/docs/rest/v6/workspace.rst

```diff
@@ -0,0 +1,512 @@
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| .title                   | String            | The human readable display name of the workspace.                        |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| .description             | String            | A longer description of the workspace.                                   |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| .base_url                | String            | The URL prefix used to access all files within the workspace.            |
+|                          |                   | This field can be null if the workspace is not web-accessible.           |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| .is_active               | Boolean           | Whether the workspace is active (false once workspace is archived).      |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| .created                 | ISO-8601 Datetime | When the associated database model was initially created.                |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| .archived                | ISO-8601 Datetime | When the workspace was archived (no longer active).                      |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| .last_modified           | ISO-8601 Datetime | When the associated database model was last saved.                       |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+
+
+.. _rest_v6_workspace_create:
+
+v6 Create Workspace
+-------------------
+
+**Example POST /v6/workspaces/ API call**
+
+Request: POST http://.../v6/workspaces/
+
+ .. code-block:: javascript  
+ 
+    {
+        "title": "Raw Source",
+        "description": "Raw Source Workspace",
+        "base_url": "http://host.com/rs",
+        "is_active": true,
+        "configuration": {
+            "broker": {
+                "type": "host",
+                "host_path": "/host/path"
+            }
+        }
+    }
+ 
+Response: 201 Created
+Headers:
+Location http://.../v6/workspaces/105/
+
+ .. code-block:: javascript  
+ 
+    { 
+        "id": 1, 
+        "name": "raw-source", 
+        "title": "Raw Source", 
+        "description": "Raw Source Workspace", 
+        "base_url": "http://host.com/rs", 
+        "is_active": true,
+        "created": "2015-10-05T21:26:04.855Z", 
+        "archived": null, 
```

Comment:
Rename `archived` to `deprecated` for consistency with everything else.
In that same migration, delete the `used_size` and `total_size` model fields.